cypress/e2e/tests/extensions/kubewarden/kubewarden.spec.ts

```diff
@@ -0,0 +1,391 @@
+  my-opentelemetry-operator open-telemetry/opentelemetry-operator`;
+
+    kubewardenPo.kubectlShell().openTerminal();
+    kubewardenPo.kubectlShell().executeCommand(addOTCommand, false);
+    kubewardenPo.appsPage().waitForAppToInstall('my-opentelemetry-operator');
+
+    // 5.2 - Jaeger Operator as per https://docs.kubewarden.io/next/howtos/telemetry/tracing-qs#install-jaeger
+    const addJaegerOperatorCommand = `helm repo add jaegertracing https://jaegertracing.github.io/helm-charts
+
+helm upgrade -i --wait \
+  --namespace jaeger \
+  --create-namespace \
+  --version 2.49.0 \
+  jaeger-operator jaegertracing/jaeger-operator \
+  --set rbac.clusterRole=true`;
+
+    kubewardenPo.kubectlShell().openTerminal();
+    kubewardenPo.kubectlShell().executeCommand(addJaegerOperatorCommand, false);
+    kubewardenPo.appsPage().waitForAppToInstall('jaeger-operator');
+
+    // 5.3 - Create Jaeger Resource as per https://docs.kubewarden.io/next/howtos/telemetry/tracing-qs#install-jaeger
+    const createJaegerResource = `kubectl apply -f - <<EOF
+apiVersion: jaegertracing.io/v1
+kind: Jaeger
+metadata:
+  name: my-open-telemetry
+  namespace: jaeger
+spec:
+  ingress:
+    enabled: true
+    annotations:
+      kubernetes.io/ingress.class: nginx
+EOF`;
+
+    kubewardenPo.kubectlShell().openTerminal();
+    kubewardenPo.kubectlShell().executeCommand(createJaegerResource, false);
+    cy.wait(5000); // eslint-disable-line cypress/no-unnecessary-waiting
+    kubewardenPo.kubectlShell().closeTerminal();
+
+    // 5.4 - Enable Tracing in kubewarden operator chart
+    installedApps.goTo();
+    installedApps.waitForPage();
+
+    installedApps.list().resourceTable().sortableTable().rowActionMenuOpen('rancher-kubewarden-controller')
+      .getMenuItem('Edit/Upgrade')
+      .click();
+    kubewardenPo.chartInstallPage().nextPage();
+    new TabbedPo().clickTabWithSelector('[data-testid="btn-Telemetry"]'); // go to Telemetry tab
+    kubewardenPo.chartInstallPage().getCheckboxByLabel('Enable Tracing').set();
+    kubewardenPo.chartInstallPage().getCheckboxByLabel('Jaeger endpoint insecure TLS configuration').set();
+    kubewardenPo.chartInstallPage().installChart();
+    kubewardenPo.appsPage().waitForUpgradeAndCloseTerminal(MAIN_EXTENSION_CHART_UPGRADE, 60000);
+
+    // 6 - ***** Install Metrics part *****
+
+    // 6.1 - Install Rancher Monitoring app
+    // needs to be a manual visit as the goTo + createPath were adding unnecessary chars that made url assertion fail
+    cy.visit('c/local/apps/charts/install?repo-type=cluster&repo=rancher-charts&chart=rancher-monitoring');
+    kubewardenPo.chartInstallPage().waitForChartPage('rancher-charts', 'rancher-monitoring');
+    // monitoring chart install page takes a bit to come up. The search for "rancherMonitoringInstallIntoProjectSelect"
+    // might fail because of this. Let's give it a time buffer here
+    cy.wait(10000); // eslint-disable-line cypress/no-unnecessary-waiting
```

Comment:
there should be other ways to check for screen content is available, applies to all the cy.wait(10000)'s on page